includes/type/object/class-customer-type.php

```diff
@@ -115,11 +115,7 @@ public static function register() {
 						'type'        => 'String',
 						'description' => __( 'A JWT token that can be used in future requests to for WooCommerce session identification', 'wp-graphql-woocommerce' ),
 						'resolve'     => function( $source ) {
-							if ( \get_current_user_id() === $source->ID ) {
 								return apply_filters( 'graphql_customer_session_token', \WC()->session->build_token() );
-							}
-
-							return null;
```

Comment:
Why not simply change the conditional to evaluate whether the `$source->ID` is a WooCommerce guest user ID. They tend to be random string. 🤔

Reply:
ok I will take a look into it, right now the 
```
$source->ID
```
does not contain a value if the user is a guest. From a quick look at the extending type for customers, it maybe just be a scoping issue. I have to dig a bit deeper to confirm that or see why the id of a guest user isn't populating in the source variable.